ui/src/app/features/face-recognition/recognition-result/recognition-result.component.ts

```diff
@@ -60,8 +60,10 @@ export class RecognitionResultComponent implements OnDestroy {
    */
   drawCanvas(box: any, face: any) {
     const img = new Image();
+    const resultFace = face.length > 0 ? face[0] : { face_name: 'Undefined', similarity: 0 };
```

Comment:
'Undefined' as a string? if this is intentional it should come from translations